src/stream.c

```diff
@@ -35,26 +35,26 @@ size_t stream_availableBytes(const stream_t* stream)
 }
 
 // Ensures that stream has at least @len available bytes
-void stream_ensureAvailableBytes(const stream_t* stream, size_t len)
+void stream_ensureAvailableBytes(const stream_t* stream, size_t size)
 {
-	if (stream_availableBytes(stream) < len) {
+	if (stream_availableBytes(stream) < size) {
 		// Soft error -- wait for next APDU
 		THROW(ERR_NOT_ENOUGH_INPUT);
 	}
 }
 
 // Advances stream by @len
-void stream_advancePos(stream_t* stream, size_t len)
+void stream_advancePos(stream_t* stream, size_t size)
 {
 	// just in case somebody changes to signed
-	ASSERT(len >= 0);
+	ASSERT(size >= 0);
```

Comment:
while way better than `len`, `size` is still a bit disconnected with "advance". maybe `advanceBy` ?